docs/app/docs/devbox_examples/languages/nodejs.md

```diff
@@ -41,9 +36,9 @@ Other versions available include:
 
 ## Installing Global Packages
 
-In some situations, you may want to install packages using `npm install --global`. This will fail in Devbox since the Nix Store is immutable. 
+In some situations, you may want to install packages using `npm install --global`. This will fail in Devbox since the Nix Store is immutable.
 
-You can instead install these global packages by adding them to the list of packages in your `devbox.json`. For example: to add `yalc` and `pm2`: 
+You can instead install these global packages by adding them to the list of packages in your `devbox.json`. For example: to add `yalc` and `pm2`:
```

Comment:
I have no idea what the recommendation for adding yalc with version number would be. Something like `nodePackages.yalc@latest`?

Reply:
I think that's the best bet

Reply:
Just tested, and that seems to work. I can update to include the version strings